.github/issue_template.md

```diff
@@ -0,0 +1,17 @@
+<!-- < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < ☺
+v                               ✰  Thanks for creating an issue! ✰    
+☺ > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > >  -->
+
+## Background
+*this is where you can provide code examples or context (e.g. past PR's)*
+*(for example)*
+- *The following PRs contributed to...*
+
+## Suggested Design
+*here, you can put concrete steps as to what to do next*
+*(for example)*
+- *create test file in...*
+## Criteria
```

Comment:
nit: just to be precise that we are referring to a specific concept in Agile: https://www.productplan.com/glossary/acceptance-criteria/
```suggestion
## Acceptance Criteria
```